project/Dependencies.scala

```diff
@@ -1,19 +1,22 @@
-import sbt._
+import sbt.*
 
 object Dependencies {
 
-  val `config-tools`       = "com.evolutiongaming"        %% "config-tools"       % "1.0.4"
-  val `future-helper`      = "com.evolutiongaming"        %% "future-helper"      % "1.0.6"
-  val sequentially         = "com.evolutiongaming"        %% "sequentially"       % "1.1.4"
-  val `akka-serialization` = "com.evolutiongaming"        %% "akka-serialization" % "1.0.4"
-  val nel                  = "com.evolutiongaming"        %% "nel"                % "1.3.4"
-  val `safe-actor`         = "com.evolutiongaming"        %% "safe-actor"         % "3.0.0"
-  val `scala-tools`        = "com.evolutiongaming"        %% "scala-tools"        % "3.0.5"
-  val scalatest            = "org.scalatest"              %% "scalatest"          % "3.2.14"
+  val `config-tools`       = "com.evolutiongaming"        %% "config-tools"       % "1.0.5"
+  val `future-helper`      = "com.evolutiongaming"        %% "future-helper"      % "1.0.7"
+  val sequentially         = "com.evolutiongaming"        %% "sequentially"       % "1.2.0"
+  val `akka-serialization` = "com.evolutiongaming"        %% "akka-serialization" % "1.1.0"
+  val nel                  = "com.evolutiongaming"        %% "nel"                % "1.3.5"
+  val `safe-actor`         = "com.evolutiongaming"        %% "safe-actor"         % "3.1.0"
+  val `scala-tools`        = "com.evolutiongaming"        %% "scala-tools"        % "3.0.6"
+  val scalatest            = "org.scalatest"              %% "scalatest"          % "3.2.19"
   val `scala-logging`      = "com.typesafe.scala-logging" %% "scala-logging"      % "3.9.5"
+  val `scodec-bits`        = "org.scodec"                 %% "scodec-bits"        % "1.2.1"
```

Comment:
Not sure if it's a good idea to update scodec-bits from 1.1 to 1.2. It's not uncommon for Scala serialization libraries to change serialization format when they bump minor version.

Reply:
scodec-bits provides building block for making serializers, like "write me a byte" so it shouldn't break. But if you insist I can try to introduce binary format compatibility tests.